src/HearThis/UI/ToolStripColorArrowRenderer.cs

```diff
@@ -49,5 +49,30 @@ protected override void OnRenderMenuItemBackground(ToolStripItemRenderEventArgs
 			}
 			base.OnRenderMenuItemBackground(e);
 		}
+
+		/// <summary>
+		/// Without this we get no separators (or perhaps their default color matches the background?)
+		/// </summary>
+		/// <param name="e"></param>
+		protected override void OnRenderSeparator(ToolStripSeparatorRenderEventArgs e)
```

Comment:
I'm fine with this, and you can merge, but I wonder if it is necessary to draw a separator at all? Could instead the items that are being separated just have some margin when they are painted?